HelpSource/Classes/Crackle.schelp

```diff
@@ -32,5 +32,7 @@ code::
 
 // modulate chaos parameter
 { Crackle.ar(Line.kr(1.0, 2.0, 3), 0.5, 0.5) }.play;
+
+{ Crackle.ar([LFDNoise0, LFDNoise1, LFDNoise3, LFDClipNoise].choose.kr([rrand(0.1,10), rrand(0.1,10)], mul: 0.7, add: 0.3), 0.5, 0.5) }.play;
```

Comment:
On 5.Jun 2016, at 22:30 , Tiago Morais Morgado <notifications@github.com> wrote:

> In HelpSource/Classes/Crackle.schelp:
> 
> > @@ -32,5 +32,7 @@ code::
> >  
> >  // modulate chaos parameter
> >  { Crackle.ar(Line.kr(1.0, 2.0, 3), 0.5, 0.5) }.play;
> > +
> > +{ Crackle.ar([LFDNoise0, LFDNoise1, LFDNoise3, LFDClipNoise].choose.kr([rrand(0.1,10), rrand(0.1,10)], mul: 0.7, add: 0.3), 0.5, 0.5) }.play;
> 
> for me it works

Sorry, but it not only doesn�t demonstrate the idea of crackle but also is dangerous for speakers and hearing.
Here it completely overloads the audio path.